packages/web-components/fast-foundation/src/menu-item/menu-item.ts

```diff
@@ -130,6 +130,16 @@ export class FASTMenuItem extends FASTElement {
         }
     }
 
+    /**
+     * The hidden attribute.
+     *
+     * @public
+     * @remarks
+     * HTML Attribute: hidden
+     */
+    @attr({ mode: "boolean" })
+    public hidden: boolean;
+
```

Comment:
Just a note/nit - *I'm aware* of why this is here, but it gets lost in the PR. While "related" to this PR this seems like one that could've come in prior.